### PR TITLE
Improving Test Coverage for Util Module

### PR DESCRIPTION
**Pull request by piyush1594**
Added Test for Configs Class
#1143

### CODE REVIEW

**Comment by piyush1594** [comment]
retest this

**Comment by hrishin** [comment]
retest this

File: core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    public void simpleTest(){
+        result = commandline.translateCommandline("cd /tmp");
+        expected.clear();
+        expected.add("cd");
+        expected.add("/tmp");
+        assertEquals(expected,result);
+    }
+
+    @Test
+    public void simpleTestSecond(){
```
Comment (hrishin) [inline core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java]: give a meaningful name for the test. simpleCommandTest(), simpleEchoTest()

File: core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        result = commandline.translateCommandline("echo \"Hello! World\"");
+        expected.clear();
+        expected.add("echo");
+        expected.add("Hello! World");
```
Comment (hrishin) [inline core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java]: the expectation should set before `commandline.translateCommandline("echo \"Hello! World\"");`

File: core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    public void simpleTestFourth(){
+        result = commandline.
+                translateCommandline("echo \"Hello! World \'Hello Java Folks\'\"");
+        expected.clear();
+        expected.add("echo");
+        expected.add("Hello! World \'Hello Java Folks\'");
+        assertEquals(expected,result);
+    }
+
+    @Test
```
Comment (hrishin) [inline core/src/test/java/io/fabric8/maven/core/util/CommandlineTest.java]: use the expected property for @test instead of having try-catch blocks

**Comment by hrishin** [comment]
@piyush1594 Thanks a lot :+1:  this looks good now.

**Pull request merged**